src/set_theory/cardinal_ordinal.lean

```diff
@@ -905,3 +906,25 @@ le_refl _
 end bit
 
 end cardinal
+
+lemma not_injective_of_ordinal {α : Type u} (f : ordinal.{u} → α) :
+  ¬ function.injective f :=
+begin
+  let g : ordinal.{u} → ulift.{u+1} α := λ o, ulift.up (f o),
+  suffices : ¬ function.injective g,
+  { intro hf, exact this (equiv.ulift.symm.injective.comp hf) },
+  intro hg,
+  replace hg := cardinal.mk_le_of_injective hg,
+  rw ← cardinal.lift_mk at hg,
+  have := hg.trans_lt (cardinal.lift_lt_univ _),
+  rw cardinal.univ_id at this,
+  exact lt_irrefl _ this
+end
+
+lemma not_injective_of_ordinal' {α : Type v} [small.{u} α] (f : ordinal.{u} → α) :
```

Comment:
```suggestion
lemma not_injective_of_ordinal_of_small {α : Type v} [small.{u} α] (f : ordinal.{u} → α) :
```
I have a newfound dislike of primed names. :-) Not particularly concerned either way here.